serial.go

```diff
@@ -137,10 +137,6 @@ func (sp *SerialPortList) List() {
 
 // discoverLoop periodically update the list of ports found
 func discoverLoop() {
-	serialPorts.Mu.Lock()
-	serialPorts.Ports = make([]SpPortItem, 0)
-	serialPorts.Mu.Unlock()
```

Comment:
Actually, this was needed because when encoded into JSON the result is `[]` instead of `null`.

It will be completely removed in the next commits, so I didn't bother to clean up this.